core/controllers/resources.py

```diff
@@ -209,6 +208,9 @@ def put(self) -> None:
         promo_bar_enabled_value = self.normalized_payload['promo_bar_enabled']
         promo_bar_message_value = self.normalized_payload['promo_bar_message']
 
+        assert isinstance(promo_bar_enabled_value, bool)
```

Comment:
This should be tested as part of the schema handler validation (using HANDLER_ARGS_SCHEMAS above). Can you check what other handlers do and make sure this is consistent with that?

Reply:
Right, we have schemas present in the `HANDLER_ARGS_SCHEMAS` so I have removed this check.